app/Models/Search.php

```diff
@@ -188,6 +188,34 @@ private function parseTagsSeach($input) {
 	}
 
 	/**
+	 * Parse the search string to find search values.
+	 * Every word is a distinct search value, except when using a delimiter.
+	 * Supported delimiters are single quote (') and double quotes (").
+	 *
+	 * @param string $input
+	 * @return string
+	 */
+	private function parseSearch($input) {
+		$input = $this->cleanSearch($input);
+		if (strcmp($input, '') == 0) {
+			return;
+		}
+		if (preg_match_all('/(?P<delim>[\'"])(?P<search>.*)(?P=delim)/U', $input, $matches)) {
```

Comment:
I have not tested yet, but just by reading quickly, isn't this regular expression greedy? I think it should search for the shortest `"..."`, not the longest `"..." ... "..."`. A suggestion is to replace the middle mask with `(?P<search>*?)`